suse-x64/Dockerfile

```diff
@@ -157,6 +158,28 @@ RUN chmod +x /build-gcc.sh \
     && /build-gcc.sh \
     && rm /build-gcc.sh
 
+RUN curl -sL -o /tmp/rpm-4.14.3.tar.bz2 http://ftp.rpm.org/releases/rpm-4.14.x/rpm-4.14.3.tar.bz2 \
+    && cd /tmp \
+    && echo "13711268181a6e201eb9d4eb06384a7fcc42dcc6c9057a62c53472cfc5ba44b5  rpm-4.14.3.tar.bz2" | sha256sum --check \
+    && cp /usr/include/db4/db.h /usr/include/db.h \
+    && tar -xjf /tmp/rpm-4.14.3.tar.bz2 \
+    && cd rpm-4.14.3 \
+    && curl -sL -o db-4.5.20.tar.gz http://download.oracle.com/berkeley-db/db-4.5.20.tar.gz \
+    && echo "f52cd5cea899823dd200d56556f70b33c55e48a33bb7b65ee128968dc10ca82d  db-4.5.20.tar.gz" | sha256sum --check \
+    && tar xzf db-4.5.20.tar.gz \
+    && ln -s db-4.5.20 db \
+    && cd db/build_unix/ \
+    && ../../configure --with-posixmutexes  --without-lua CPPFLAGS="-I/usr/include/nspr4 -I/usr/include/nss3 -I/usr/include/db4/"\
+    && make \
+    && make install \
+    && cd / \
+    && echo "/usr/local/lib/" >> /etc/ld.so.conf \
+    && ldconfig
```

Comment:
What is the purpose of the two above lines? AFAICS `/usr/local/lib` already is in `/etc/ld.so.conf` in the base image.

Reply:
Interesting, in the base image, I'm getting
```
# ld --verbose | grep SEARCH
SEARCH_DIR("/usr/x86_64-suse-linux/lib64"); SEARCH_DIR("/usr/lib64"); SEARCH_DIR("/usr/local/lib64"); SEARCH_DIR("/lib64"); SEARCH_DIR("/usr/x86_64-suse-linux/lib"); SEARCH_DIR("/usr/local/lib"); SEARCH_DIR("/lib"); SEARCH_DIR("/usr/lib");
```
I'm wondering if there's something that changes that during the image build. I can't see anything that would do that from brief look :thinking:

Reply:
Uhm I don't know can't find anything that changes it 🤔 
But without it doesn't find it and with it does do I keep it this way ?